### PR TITLE
Sort imports

### PR DESCRIPTION
**Pull request by MatteoStifano**
Sort imports in the e2e tests' module

### CODE REVIEW

**Comment by BenediktSeidl** [comment]
if you want, you can extend this to the whole codebase.
ideally you could extend the ci so unsorted imports would fail. otherwise it will degrade over time.

**Comment by MatteoStifano** [comment]
@BenediktSeidl shall I introduce the sorting in the `prettier` command?

File: .prettierrc.js
Comment (PhilippSiegmantel) [inline .prettierrc.js:8]: I don't think this has any effect.

Should probably be something along the lines of
```
module.exports = {
  ...require('./node_modules/@grafana/toolkit/src/config/prettier.plugin.config.json'),
}
module.exports.importOrder = ['^components/(.*)$', '^[./]']
...
```

**Review by PhilippSiegmantel — CHANGES_REQUESTED** [review]
Please modify .prettierrc.js as noted in the comment and rerun prettier.

File: .prettierrc.js
Comment (MatteoStifano) [reply .prettierrc.js]: Done.

**Comment by BenediktSeidl** [comment]
let's please wait until we've merged #102, because it's super easy to rerun the prettier, but would mean some serious  work to rebase the whole chain.

**Comment by MatteoStifano** [comment]
@BenediktSeidl no problem

**Review by BenediktSeidl — APPROVED** [review]

File: .prettierrc.js
Comment (PhilippSiegmantel) [inline .prettierrc.js:6]: `module.exports` is a normal variable, there is no need for the parens here. Should look like this:
```
module.exports = {
  ...require('./node_modules/@grafana/toolkit/src/config/prettier.plugin.config.json'),
};
module.exports.importOrder = ['^components/(.*)$', '^[./]'];
module.exports.importOrderSeparation = true;
module.exports.importOrderSortSpecifiers = true;
```

**Review by PhilippSiegmantel — CHANGES_REQUESTED** [review]
Please don't forget to rerun prettier after changing the rc.js

File: .prettierrc.js
Comment (MatteoStifano) [reply .prettierrc.js]: It appears that by running `yarn prettier --write ./` from the main folder, also the configuration file itself is modified. Those parentheses are due to the `prettier` formatting.

The reason is that after each variable declaration, typescript expects a semicolon, not a comma. By introducing semicolons, the issue is fixed.

This does not affect the format of all other files.

Comment (PhilippSiegmantel) [reply .prettierrc.js]: Looks great!

**Review by PhilippSiegmantel — APPROVED** [review]

**Pull request merged**
